consensus/checks.go

```diff
@@ -146,7 +164,7 @@ func (consensus *Consensus) onPreparedSanityChecks(
 			Msg("[OnPrepared] BlockHash not match")
 		return false
 	}
-	if consensus.current.Mode() == Normal {
+	if consensus.current.Mode() == Normal || consensus.current.Mode() == Syncing {
```

Comment:
@chaosma does this change look good to this issue? https://github.com/harmony-one/security-audit/issues/14

Reply:
@rlan35  In this case, it seems that we could add sanity check for all modes including view change. Also, according to the security audit, we could store the received message according to hash of the content instead of address.